data/json/recipes/recipe_others.json

```diff
@@ -7003,5 +7003,41 @@
   "components": [
     [ [ "splinter", 5 ], [ "2x4", 1 ], [ "stick", 1 ] ]
   ]
+},{
+  "type" : "recipe",
+  "result": "link",
+  "id_suffix": "fabrication",
+  "category": "CC_OTHER",
+  "subcategory": "CSC_OTHER_MATERIALS",
+  "skill_used": "fabrication",
+  "difficulty": 2,
+  "time": 5000,
+  "book_learn": [["textbook_armwest", 1] , ["textbook_fabrication", 1] , ["recipe_melee", 1]],
+  "tools": [
+    [ [ "pliers", -1 ], [ "multitool", -1 ] ],
+    [ [ "boltcutters", -1 ], [ "toolset", -1 ] ],
+    [ ["cu_pipe", -1] ]
+  ],
+  "components": [ [ [ "wire", 1 ] ] ]
+},{
+  "type" : "recipe",
+  "result": "link_sheet",
```

Comment:
Could probably be reversible.

Reply:
The links are riveted closed, rather than butted. They can't be disconnected and re-used without compromising their structural integrity (creating a second rivet/hole/stress concentration). A certain fraction of them might be salvageable by cutting some rings in a way that releases others, but it would likely be much easier for the player to simply make more rings rather than attempt to recycle the chainmail.

Reply:
Agreed that if it's reversible it needs to be lossy, that also brings up repair, but that can wait for later.